src/App.js

```diff
@@ -198,7 +217,11 @@ class App extends Component {
             {renderChatBubble}
             <RootContainer>
               <Switch>
-                <Route exact path="/" component={EnhancedBrowseSkill} />
+                {this.renderSkillsPage ? (
```

Comment:
Should we redirect to a separate route having control page? By this approach, the current home page won't be there at all.

Reply:
I discussed this with alok, the check_ap point returns false after the smart speaker setup. So, once we set up the smart speaker, it would return false and go to current homepage